internal/jujuapi/modelmanager_test.go

```diff
@@ -221,23 +221,35 @@ func (s *modelManagerSuite) TestListModelSummariesWithoutControllerUUIDMasking(c
 }
 
 func (s *modelManagerSuite) TestListModels(c *gc.C) {
-	conn := s.open(c, nil, "bob")
+	conn := s.open(c, nil, "charlie@canonical.com")
 	defer conn.Close()
 
 	client := modelmanager.NewClient(conn)
-	models, err := client.ListModels("bob")
+	models, err := client.ListModels("charlie@canonical.com")
```

Comment:
since, we are not really filtering based of this email when using the juju client. Can we use another more expressive email?
Like: anyuser@canonical.com

Reply:
i've taken a look at the suite, i didn't realise charlie had access to these two models.
Ignore this comment.